plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/OAuth2SecurityConfig.java

```diff
@@ -27,6 +28,7 @@ public class OAuth2SecurityConfig
     private String scope;
     private String token;
     private URI serverUri;
+    private boolean tokenRefreshEnabled = OAuth2Properties.TOKEN_REFRESH_ENABLED_DEFAULT;
```

Comment:
When do we want to disable this property?

Reply:
In case the token is managed from the outside. For example it is passed via `X-Trino-Extra-Credential` header

Reply:
@ebyhr any other concerns?)